main.py

```diff
@@ -393,33 +422,40 @@ def _desktop_notification(self, title: str, message: str):
 
 
 def main():
-    print("vaccipy 1.0\n")
+    print("vaccipy - Automatische Terminbuchung für den Corona Impfterminservice")
 
     # Check, ob die Datei "kontaktdaten.json" existiert
     kontaktdaten_path = os.path.join(PATH, "kontaktdaten.json")
     kontaktdaten_erstellen = True
     if os.path.isfile(kontaktdaten_path):
-        daten_laden = input(
-            "Sollen die vorhandene Daten aus 'kontaktdaten.json' geladen werden (y/n)?: ").lower()
-        if daten_laden != "n":
+        daten_laden = input("\n> Sollen die vorhandene Daten aus 'kontaktdaten.json' "
+                            "geladen werden (y/n)?: ").lower()
+        if daten_laden.lower() != "n":
             kontaktdaten_erstellen = False
 
     if kontaktdaten_erstellen:
-        print("Bitte trage zunächst deinen Impfcode und deine Kontaktdaten ein.\n"
+        print("\nBitte trage zunächst deinen Impfcode und deine Kontaktdaten ein.\n"
               "Die Daten werden anschließend lokal in der Datei 'kontaktdaten.json' abgelegt.\n"
               "Du musst sie zukünftig nicht mehr eintragen.\n")
-        code = input("Code: ")
-        plz = input("PLZ des Impfzentrums: ")
-
-        anrede = input("Anrede (Frau/Herr/...): ")
-        vorname = input("Vorname: ")
-        nachname = input("Nachname: ")
-        strasse = input("Strasse: ")
-        hausnummer = input("Hausnummer: ")
-        wohnort_plz = input("PLZ des Wohnorts: ")
-        wohnort = input("Wohnort: ")
-        telefonnummer = input("Telefonnummer: +49")
-        mail = input("Mail: ")
+        code = input("> Code: ")
+        print(
+            "\nMit einem Code kann in mehreren Impfzentren gleichzeitig nach einem Termin gesucht werden.\n"
+            "Eine Übersicht über die Gruppierung der Impfzentren findest du hier:\n"
+            "https://github.com/iamnotturner/vaccipy/wiki/Ein-Code-fuer-mehrere-Impfzentren\n\n"
+            "Trage nun die PLZ deines Impfzentrums ein. Für mehrere Impfzentren die PLZ's kommagetrennt nacheinander.\n"
+            "Beispiel: 68163, 69124, 69469\n")
+        plz_impfzentren = input("> PLZ's der Impfzentren: ")
+        plz_impfzentren = [plz.strip() for plz in plz_impfzentren.split(",")]
```

Comment:
Evtl sollten wir hier in einer Loop die Impfzentren abfragen, bis man nicht mehr möchte. Wenn jemand 79123 , 79120 schreibt, wird das Leerzeichen nicht entfernt.
Die loop sollte ein set an impfzentren erweitern, um Dopplungen zu vermeiden.

Reply:
PLZ Duplikate werden jetzt entfernt

habe mich gegen die iterative PLZ-Eingabe entschieden, weil ich das nicht ganz so handy finde.
persönlich fände ich das eher nervig, und irgendwie auch kompliziert, weil man den Loop dann mit Q abbrechen müsste oder so - eine kommagetrennte Eingabe der PLZs sollte klar gehen